### ISSUE TITLE
[torrentday] Please reenter captcha (Config)

### ISSUE THREAD

**Issue by CollinNL** [Jackett/Jackett#1992]
Unable to add 'Torrentday' as indexer, it gives this error:
```
An error occured while updating this indexer
Please reenter captcha

```
Jackett version:  
0.8.285.0 Win 10

**Comment by garfield69** [comment]
The problem is simply that you did not solve the captcha correctly.
So try adding the definition again.

**Comment by kaso17** [comment]
working fine for me, if it's still a problem, please provide a full log

**Comment by CollinNL** [comment]
@garfield69 Nope, it's successful but it still gives the error.

**Comment by bicey5** [comment]
Also having this issue. Trying to set Jackett up for use with Sonarr & Radarr on a new Windows 10 Pro PC.

![jackett recaptcha issue](https://user-images.githubusercontent.com/33079798/31999827-aa4331e0-b962-11e7-9cc8-e8006aad0f1d.PNG)

**Comment by kaso17** [comment]
Google apparently just fixed the 127.0.0.1 loophole for recaptchas.
With the next release recaptcha support will be disabled.
Until then use localhost instead of 127.0.0.1 to access jackett.
